tests/framework/installer/ceph_helm_installer.go

```diff
@@ -79,6 +79,18 @@ func (h *CephInstaller) CreateRookCephClusterViaHelm(values map[string]interface
 		"enabled": true,
 		"image":   "rook/ceph:" + LocalBuildTag,
 	}
+	values["ingress"] = map[string]interface{}{
+		"dashboard": map[string]interface{}{
+			"annotations": map[string]interface{}{
+				"kubernetes.io/ingress-class":                "nginx",
+				"nginx.ingress.kubernetes.io/rewrite-target": "/ceph-dashboard/$2",
+			},
+			"host": map[string]interface{}{
+				"name": "example.rook.io",
+				"path": "/ceph(/|$)(.*)",
+			},
+		},
+	}
```

Comment:
Since we are going through the effort to set up ingress in the test, is it possible to verify the ingress is working later in the test? Maybe we could do a `curl` to the endpoint and see that we get an expected Ceph dashboard result, for example?

Reply:
I'm not super sure we will be able to curl the endpoint because it might not actually be exposed using a useful url, but I can investigate.

I will add a test that checks that the ingress is healthy though.